src/module.ml

```diff
@@ -175,14 +177,24 @@ let wrapped_compat t =
       Some (
         { syntax = OCaml
         ; path =
-          Path.L.relative (dir t)
-            [ ".wrapped_compat"
-            ; Name.to_string t.name ^ ".ml-gen"
-            ]
+            Path.L.relative (Option.value_exn (src_dir t))
```

Comment:
This cannot raise because we disallow unwrapped mode for virtual libraries. This fact is not obvious, so we should mention it in a comment here.

Reply:
Actually, it's because we disallow wrapped with compat